lib/src/key_pair/key_pair.ts

```diff
@@ -1,6 +1,6 @@
 import ow from 'ow';
 
-const native = require('../../native');
+const native = require('../../../native');
 
 export default class KeyPair {
     private innerPublicKey?: Buffer;
```

Comment:
I am confused that why add `inner` prefix for KeyPair class here.

Reply:
I have later provide two getter methods `publicKey` and `privateKey`. I do this so to avoid collision of name between the method and instance properties.